docs/howto/account-billing/change-billing-data.md

```diff
@@ -0,0 +1,30 @@
+# Changing your account billing data
+
+Via [{{gui}}](https://{{gui_domain}}) you can change the contact
+person, address, company name, and purchase order number associated
+with your account.
+
+To get started, navigate to <https://{{gui_domain}}>.  Log in and
+click on the _Profile_ button at the top right.
+
+![Orange avatar person with white
+background.](assets/picture-profile.png)
+
+You will now see the _Account settings_ page. Choose the _Customer
+Info_ tab, in which you can change and manage your customer
+information.
+
+![The "Customer Info" tab with a green "Update" button to complete the
+changes](assets/picture-changecustomerinfo2.png)
+
+Finish your changes by clicking the green _Update_ button at the
+bottom.
+
+## Changing an account's organization number
+
+You can not change your organization number (for business accounts) or
```

Comment:
As in most cases, "can not" should be written as "cannot".

Reply:
Done.